pkg/handlers/mount.go

```diff
@@ -110,11 +119,69 @@ func Mount(allowedFilesystems map[string]struct{}) registry.HandlerFunc {
 			return registry.HandlerResultContinue()
 		}
 
+		var options string
+		if req.Data.Args[4] != 0/* NULL */ && filesystem != "proc" && filesystem != "sysfs" {
```

Comment:
I guess this excludes proc and sysfs for security reasons? Do we want to update the docs to reflect this there? Or even better, return an error where we parse the allowed fs map if these are used?

Reply:
Fair, I'll read the procmount KEP later. Do you mind then adding a comment to say we ignore the options thing for those fs because there are better alternatives like that KEP or something like that?

Reply:
I was more worried about the behaviour change, actually now I've checked it, I think it's fine to allow options for proc (could allow setting [`hidepid`](https://docs.kernel.org/filesystems/proc.html#mount-options)). sysfs doesn't have any options, so I've not allowed that to avoid any future surprises.